root/base.tx

```diff
@@ -115,10 +115,17 @@
                       </div>
                   </div>
                 </form>
+                %% if ! $current['/'] {
+                    <li class="icon-slidepanel visible-xs visible-sm">
+                      <button data-toggle="slidepanel" data-target=".slidepanel">
+                        <i class="fa fa-bars button-fa-icon"></i>
+                      </button>
+                    </li>
+                %% }
                 <form action="/account/logout" method="POST" id="metacpan-logout"></form>
                 <li class="dropdown logged_in" style="display: none;">
                     <button type="button" class="dropdown-toggle" data-toggle="dropdown">
-                      <i class="fa fa-user" aria-hidden="true"></i>
+                      <img class="logged-in-gravatar" src="[% gravatar_image($author) %]">
```

Comment:
I'm a bit stuck on figuring out the right syntax for `[% gravatar_image($author) %]`. I'm not sure what to replace $author with in order to get the gravatar of the logged in user. I temporarily added this in just to be able to visualize what the gravatar _would_ look like.

Reply:
Right. So, I think `$author` will refer to the author of a module on a page that's being viewed rather than the profile of a logged-in user (who in some cases may not even be an author).

I think what we might want to do is update the login logic so that the user's gravatar_url is always available in the session after login or have some other way to make it available for every page request. So, this particular fix might need to live in a different branch until that back end code change has taken place.

Reply:
We'll need to add the appropriate data to the login_status request. It's probably best to put it in a separate branch/PR.

One challenge here is that for authors, we use the `@cpan.org` email alias for the gravatar. Not all logged in users have an associated PAUSE account. So we'll still need a default user icon.

Reply:
That makes sense. For now, I added back in the red icon color.

<img width="1511" alt="Screen Shot 2022-10-02 at 8 16 20 AM" src="https://user-images.githubusercontent.com/52248962/193473897-65dac329-31a8-4539-9d6b-aac7acbf27fe.png">
<img width="589" alt="Screen Shot 2022-10-02 at 8 16 56 AM" src="https://user-images.githubusercontent.com/52248962/193473898-e894f9f5-8281-4cd9-ba77-eb224818dcaa.png">